src/ui-test/tests/variables.test.ts

```diff
@@ -34,7 +35,7 @@ import {
     isVersionNewer,
     DEBUGGER_ATTACHED_MESSAGE,
 } from '../utils';
-import { RESOURCES_DIR, VARIABLESTEST_JAVA } from '../variables';
+import { RESOURCES_DIR, VARIABLESTEST_YAML } from '../variables';
```

Comment:
why was the change from` Java DSL` to `YAML DSL` needed?

Reply:
I think it's better to have YAML as it's preferred format. There is no difference in functionality.